TriangleNet_Engine/TriangleNet_Engine.csproj

```diff
@@ -70,6 +70,9 @@
       <HintPath>..\..\BHoM\Build\Graphics_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Quantities_oM">
+      <HintPath>..\..\BHoM\Build\Quantities_oM.dll</HintPath>
```

Comment:
Copy local false!

Reply:
Probably needs doing on other references - this wasn't done last week I think because of the other PR that @alelom had open would have clashed.

Reply:
Ok, will run my script on this to get all sorted.